plugins/woocommerce-beta-tester/includes/class-wc-beta-tester-admin-menus.php

```diff
@@ -153,6 +153,10 @@ private function get_plugins_ssr_info( $active_plugins, $untested_plugins ) {
 	 * @return string
 	 */
 	protected function construct_ssr() {
+		if ( ! class_exists( 'WC' ) ) {
```

Comment:
This fixes a bug that exists between deactivating one version of WC and activating another. the WC class doesn't exist sometimes and so it crashes. I think a future enhancement would to be remove the dependency on WC class as I have found some good ways to determine WC version without this call in the live branches code that we could use.

Reply:
A comment in the code saying this could be helpful for anyone coming across this.

Reply:
Fixed.